### PR TITLE
RC: Vercel integration

### PR DESCRIPTION
**Pull request by kaitlynmichael**
DOC-4151

### CODE REVIEW

File: content/operate/rc/cloud-integrations/vercel.md [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+categories:
+- docs
+- operate
+- rc
+description: Shows how to create a Redis Cloud database with Vercel integration.
+hideListLinks: true
+weight: 66
+---
+
+[Redis Cloud is now integrated with Vercel](https://vercel.com/redis-cloud/~/integrations/products/redis-cloud), allowing you to create a new Redis database from your Vercel account and connect it to your Vercel project(s).
```
Comment (cmilesb) [inline content/operate/rc/cloud-integrations/vercel.md:14]:
```suggestion
The [Redis Cloud Vercel integration](https://vercel.com/redis-cloud/~/integrations/products/redis-cloud) lets you create a new Redis database from your Vercel account and connect it to your Vercel project(s).
```

File: content/operate/rc/cloud-integrations/vercel.md [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+
+1. Select **Create**.
+
+## Connect to your database
+
+After creation, you will see your database details. After provisioning is complete, the status will change from `Initializing` to `Available` (you may need to refresh your browser).
+
+{{<image filename="/images/rc/vercel-db-detail.png" alt="Vercel database details">}}
+
+You can use the connection string shown under **Quickstart** to [connect to your database]({{<relref "operate/rc/rc-quickstart/#connect-to-a-database">}}).
```
Comment (cmilesb) [inline content/operate/rc/cloud-integrations/vercel.md:44]:
```suggestion
You can use the connection string shown under **Quickstart** to [connect to your database]({{<relref "operate/rc/databases/connect">}}).
```

File: content/operate/rc/cloud-integrations/vercel.md [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+
+From the database detail page, select **Open in Redis Cloud**.
+
+{{<image filename="/images/rc/vercel-db-detail.png" alt="Vercel database details">}}
+
+Your Redis Cloud account is linked to your Vercel account. All your team's Redis databases will be listed under **Databases** in Redis Cloud.
+
+Select your new database to make configuration changes such as passwords or the eviction policy.
+
+{{<note>}} The [eviction policy]({{<relref "/operate/rc/databases/configuration/data-eviction-policies/">}}) defaults to `no eviction` for new databases.  You can change this by [editing the database details]({{<relref "/operate/rc/databases/view-edit-database/">}}).{{</note>}}
```
Comment (cmilesb) [inline content/operate/rc/cloud-integrations/vercel.md:73]:
```suggestion
{{<note>}} The [eviction policy]({{<relref "/operate/rc/databases/configuration/data-eviction-policies">}}) defaults to `no eviction` for new databases.  You can change this by [editing the database details]({{<relref "/operate/rc/databases/view-edit-database">}}).{{</note>}}
```
Need to remove the last slash - the links didn't work properly in the staged preview.

File: static/images/rc/vercel-storage-tab-create-db.png
Comment (cmilesb) [inline static/images/rc/vercel-storage-tab-create-db.png:1]: I would crop out the navigation tabs here, so it just shows the heading "Storage" and the "Create Database" button.

File: content/operate/rc/cloud-integrations/vercel.md [hunk 4]
```diff
@@ -0,0 +1,73 @@
...
+
+1. Enter your database name or use the leave the automatically generated name.
+
+1. Select **Create**.
+
+## Connect to your database
+
+After creation, you will see your database details. After provisioning is complete, the status will change from `Initializing` to `Available` (you may need to refresh your browser).
+
+{{<image filename="/images/rc/vercel-db-detail.png" alt="Vercel database details">}}
```
Comment (cmilesb) [inline content/operate/rc/cloud-integrations/vercel.md:42]: I'd just show the Status/Plan/Current period/Period total portion of this image here.

File: content/operate/rc/cloud-integrations/vercel.md [hunk 5]
```diff
@@ -0,0 +1,73 @@
...
+
+More configuration options are coming soon, the plan changes, multi-zone high availability, and region selection.
+
+### Configure from Redis Cloud
+
+You can also edit your some configuration options in Redis Cloud.
+
+From the database detail page, select **Open in Redis Cloud**.
+
+{{<image filename="/images/rc/vercel-db-detail.png" alt="Vercel database details">}}
```
Comment (cmilesb) [inline content/operate/rc/cloud-integrations/vercel.md:67]: I would show the "Open in Redis Cloud" button here.

File: static/images/rc/vercel-list-connect-project.png
Comment (cmilesb) [inline static/images/rc/vercel-list-connect-project.png:1]: I'd recommend just showing the Connect Project button.

File: static/images/rc/vercel-settings.png
Comment (cmilesb) [inline static/images/rc/vercel-settings.png:1]: I don't think this is being shown anywhere. Not sure if that's intentional or not.

File: content/operate/rc/cloud-integrations/vercel.md [hunk 6]
```diff
@@ -0,0 +1,73 @@
...
+
+## Manage your database
+
+From the database details page, you can make edits to your database under **Settings**.
+
+More configuration options are coming soon, the plan changes, multi-zone high availability, and region selection.
+
+### Configure from Redis Cloud
+
+You can also edit your some configuration options in Redis Cloud.
```
Comment (cmilesb) [inline content/operate/rc/cloud-integrations/vercel.md:63]:
```suggestion
You can also edit some configuration options in Redis Cloud.
```

File: content/operate/rc/cloud-integrations/vercel.md [hunk 7]
```diff
@@ -0,0 +1,73 @@
...
+1. Select **Connect Project**.
+    {{<image filename="/images/rc/vercel-list-connect-project.png" alt="List databases to connect project">}}
+1. Choose your project and environments and select **Connect**.
+    {{<image filename="/images/rc/vercel-connect-project.png" alt="Connect project">}}
+
+## Manage your database
+
+From the database details page, you can make edits to your database under **Settings**.
+
+More configuration options are coming soon, the plan changes, multi-zone high availability, and region selection.
```
Comment (cmilesb) [inline content/operate/rc/cloud-integrations/vercel.md:59]:
```suggestion
More configuration options are coming soon, including plan changes, multi-zone high availability, and region selection.
```
I'm not sure what this sentence is referring to - Is this list of configuration options what you can currently configure, or is it the list of options you'll be able to configure soon?This suggestion is for the latter.

**Review by cmilesb — CHANGES_REQUESTED** [review]
Looks great so far! Just have a few suggestions/questions.

File: static/images/rc/vercel-storage-tab-create-db.png
Comment (kaitlynmichael) [reply static/images/rc/vercel-storage-tab-create-db.png:1]: Fixed

File: content/operate/rc/cloud-integrations/vercel.md (hunk 5 above)
Comment (kaitlynmichael) [reply content/operate/rc/cloud-integrations/vercel.md]: Fixed

File: static/images/rc/vercel-settings.png
Comment (kaitlynmichael) [reply static/images/rc/vercel-settings.png:1]: Removed

File: static/images/rc/vercel-list-connect-project.png
Comment (kaitlynmichael) [reply static/images/rc/vercel-list-connect-project.png:1]: Fixed

File: content/operate/rc/cloud-integrations/vercel.md [hunk 8]
```diff
@@ -0,0 +1,73 @@
...
+The [Redis Cloud Vercel integration](https://vercel.com/redis-cloud/~/integrations/products/redis-cloud) lets you create a new Redis database from your Vercel account and connect it to your Vercel project(s).
+
+## Create database
+
+1. Log in to your Vercel account (or create a new one).
+
+1. Navigate to the **Storage** tab and select **Create database**.
+    {{<image filename="/images/rc/vercel-storage-create-database-button.png" alt="Storage - Create Database">}}
+
+1. Under **Storage partners**, select **View all partners**
```
Comment (cmilesb) [inline content/operate/rc/cloud-integrations/vercel.md:23]:
```suggestion
1. Under **Storage partners**, select **View all partners**.
```

File: content/operate/rc/cloud-integrations/vercel.md [hunk 9]
```diff
@@ -0,0 +1,73 @@
...
+    {{<image filename="/images/rc/vercel-storage-partners-view-all.png" alt="View all Storage Partners">}}
+
+1. Find **Redis Cloud** and select **Continue**
```
Comment (cmilesb) [inline content/operate/rc/cloud-integrations/vercel.md:26]:
```suggestion
1. Find **Redis Cloud** and select **Continue**.
```

File: content/operate/rc/cloud-integrations/vercel.md [hunk 10]
```diff
@@ -0,0 +1,73 @@
...
+
+1. In the **Create Database** dialog, select your plan and **Continue**.
+
+    {{<image filename="/images/rc/vercel-create-db-select-plan.png" alt="Create database">}}
+
+    More configuration options are coming soon, such as region selection and multi-zone high availability.
+
+1. Enter your database name or use the leave the automatically generated name.
```
Comment (cmilesb) [inline content/operate/rc/cloud-integrations/vercel.md:34]:
```suggestion
1. Enter your database name or use the automatically generated name.
```

**Review by cmilesb — APPROVED** [review]
Looks good to me! Just a few periods.

File: content/operate/rc/cloud-integrations/vercel.md [hunk 11]
```diff
@@ -0,0 +1,73 @@
...
+categories:
+- docs
+- operate
+- rc
+description: Shows how to create a Redis Cloud database with Vercel integration.
+hideListLinks: true
+weight: 66
+---
+
+The [Redis Cloud Vercel integration](https://vercel.com/redis-cloud/~/integrations/products/redis-cloud) lets you create a new Redis database from your Vercel account and connect it to your Vercel project(s).
```
Comment (cmilesb) [inline content/operate/rc/cloud-integrations/vercel.md:14]:
```suggestion
The [Redis Cloud Vercel integration](https://vercel.com/marketplace/redis-cloud) lets you create a new Redis database from your Vercel account and connect it to your Vercel project(s).
```

**Pull request merged**